daemons/compute/server/servers/server_default.go

```diff
@@ -206,11 +208,14 @@ func CreateDefaultServer(opts *ServerOptions) (*defaultServer, error) {
 }
 
 func (s *defaultServer) StartManager() error {
+	namespaceCache := make(map[string]cache.Config)
+	namespaceCache[s.namespace] = cache.Config{}
+
 	mgr, err := ctrl.NewManager(s.config, ctrl.Options{
-		Scheme:             scheme,
-		LeaderElection:     false,
-		MetricsBindAddress: "0",
-		Namespace:          s.namespace,
+		Scheme:         scheme,
+		LeaderElection: false,
+		Metrics:        metricsserver.Options{BindAddress: "0"},
+		Cache:          cache.Options{DefaultNamespaces: namespaceCache},
```

Comment:
I ended up doing this for remora:

```
cache.Options{DefaultNamespaces: map[string]cache.Config{
    s.namespace: {},
}}
```

Then you could get rid of the lines above.

Reply:
I'll try that.